terraform/production/main.tf

```diff
@@ -185,20 +193,23 @@ module "database_migration" {
 
   count = local.create_db_migration_infra ? 1 : 0
 
-  prefix                         = local.prefix
-  cloudfoundry_service           = "dluhc-core-production-postgres"
-  cloudfoundry_space             = "production"
-  database_connection_string_arn = module.database.rds_connection_string_arn
-  database_port                  = local.database_port
-  db_migration_task_cpu          = 4096
-  db_migration_task_memory       = 16384
-  db_security_group_id           = module.database.rds_security_group_id
-  ecr_repository_url             = "815624722760.dkr.ecr.eu-west-2.amazonaws.com/db-migration"
-  ecs_task_role_arn              = module.application.ecs_task_role_arn
-  ecs_task_execution_role_arn    = module.application.ecs_task_execution_role_arn
-  ecs_task_execution_role_name   = module.application.ecs_task_execution_role_name
-  ecs_task_ephemeral_storage     = 200 #GiB
-  vpc_id                         = module.networking.vpc_id
+  db_migration_task_cpu      = 4096
+  db_migration_task_memory   = 16384
+  ecs_task_ephemeral_storage = 200 #GiB
+
+  ecr_repository_url = "815624722760.dkr.ecr.eu-west-2.amazonaws.com/db-migration"
+
+  cloudfoundry_service = "dluhc-core-production-postgres"
+  cloudfoundry_space   = "production"
+
+  prefix                                  = local.prefix
+  database_complete_connection_string_arn = module.application.rds_complete_connection_string_arn

```

Comment:
This is the only real change here (likewise for the other envs), replacing the `database_connection_string_arn`. I tidied up the variable here so it looks worse than it is 😀.